API.md

```diff
@@ -115,6 +115,8 @@ Returns a `Promise` that resolves to [a database instance](#store-api). `name` (
 
 - `overwrite` (boolean): Overwrite an existing database (Default: `false`)
 - `replicate` (boolean): Replicate the database with peers, requires IPFS PubSub. (Default: `true`)
+- `meta` (object): An object to be added to the database manifest (learn more about the [db manifest](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#address)). Data in the manifest affects the address of the db and cannot be changed after creation. It's recommended to store smaller objects and must be serializable to json. Read the object at instance property `db.options.meta`. Default: `undefined`.
+
 ```javascript
```

Comment:
I know this isn't your job... In this PR but would you mind adding all of the above options to the JS example for posterity?

Reply:
sure, should this be in a separate pr?

Reply:
I wouldn't mind if it was in the same PR since it's doc-only